src/CommunityToolkit.Maui/Converters/ItemSelectedEventArgsConverter.shared.cs

```diff
@@ -18,6 +19,7 @@ public class ItemSelectedEventArgsConverter : ValueConverterExtension, IValueCon
 	/// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
 	/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
 	/// <returns>A <see cref="SelectedItemChangedEventArgs.SelectedItem"/> object from object of type <see cref="SelectedItemChangedEventArgs"/>.</returns>
+	[return: NotNullIfNotNull("value")]
```

Comment:
please use `nameof(value)`

Reply:
I wish I could! I hate hard-coded strings, but since `value` is a method parameter and the attribute is outside the method, we cannot use `nameof(value)` because the variable name `value` hasn't been defined yet.